File: src/config/env.js
import "dotenv/config";

export const ENV = {
  DISCORD_TOKEN: process.env.DISCORD_TOKEN,
  CLIENT_ID: process.env.CLIENT_ID,
  GUILD_ID: process.env.GUILD_ID,

  WEBHOOK_ID: process.env.WEBHOOK_ID,
  WEBHOOK_TOKEN: process.env.WEBHOOK_TOKEN,

  CHEF_ROLE_IDS: process.env.CHEF_ROLE_IDS?.split(","),
  ADMIN_ROLE_IDS: process.env.ADMIN_ROLE_IDS?.split(","),

  ORDER_CATEGORY_ID: process.env.ORDER_CATEGORY_ID,
  DISCOUNT_CHANNEL_ID: process.env.DISCOUNT_CHANNEL_ID,

  SUPABASE_URL: process.env.SUPABASE_URL,
  SUPABASE_ANON_KEY: process.env.SUPABASE_ANON_KEY
};

File: src/config/env.py
import os
from dotenv import load_dotenv

load_dotenv()

DISCORD_TOKEN = os.getenv("DISCORD_TOKEN")
CLIENT_ID = os.getenv("CLIENT_ID")
GUILD_ID = int(os.getenv("GUILD_ID"))

WEBHOOK_ID = int(os.getenv("WEBHOOK_ID"))
WEBHOOK_TOKEN = os.getenv("WEBHOOK_TOKEN")

CHEF_ROLE_IDS = [int(x) for x in os.getenv("CHEF_ROLE_IDS").split(",")]
ADMIN_ROLE_IDS = [int(x) for x in os.getenv("ADMIN_ROLE_IDS").split(",")]

ORDER_CATEGORY_ID = int(os.getenv("ORDER_CATEGORY_ID"))
DISCOUNT_CHANNEL_ID = int(os.getenv("DISCOUNT_CHANNEL_ID"))

SUPABASE_URL = os.getenv("SUPABASE_URL")
SUPABASE_ANON_KEY = os.getenv("SUPABASE_ANON_KEY")
